docs/docs/versions.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Versions
+layout: docs
+---
+
+Because the TSC library is a client for the Tableau Server REST API, you need to confirm that the version of the TSC
+library that you use is compatible with the version of the REST API used by your installation of Tableau Server.
+
+* TOC
+{:toc}
+
+## Display the REST API version
+
+To display the default version of the REST API used by TSC, run the following code:
+
+```py
+import tableauserverclient as TSC
+
+server = TSC.Server('http://SERVER_URL')
+
+print(server.version)
```

Comment:
I think we need to leave this in. Yes, it's hardcoded to 2.3 right now, but how would you know that if you ran pip install? You'd have to look at the source or run this command. This will be more useful when we have multiple releases as well.